commitizen/wrap_stdio_linux.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+if sys.platform == "linux":  # pragma: no cover
```

Comment:
The coverage of pytest is check all files.
if it is removed from wrap_stdio_linux.py,  the git hub release test for mac os is not passed.

```
---------- coverage: platform darwin, python 3.7.3-final-0 -----------
Name                                                         Stmts   Miss  Cover   Missing
------------------------------------------------------------------------------------------
:
commitizen/wrap_stdio_linux.py                                   6      2    67%   12, 16
```

Reply:
and same reason.
wrap_stdio_unix.py is checked on the linux environment.

```
---------- coverage: platform linux, python 3.8.10-final-0 -----------
Name                                                         Stmts   Miss  Cover   Missing
------------------------------------------------------------------------------------------
:
commitizen/wrap_stdio_unix.py                                   42     42     0%   1-73
```

Reply:
Got it. From my point of view, we should write cleaner code instead of just trying to match the check. I'm ok with skipping part of the coverage check if it does make the code cleaner.